_posts/2023-01-12-January-Update-Finishing-my-Igalia-CE.md

```diff
@@ -0,0 +1,250 @@
+iterations.
+
+The second side-quest was to understand why the IGT test `kms_writeback` was
+causing a kernel oops in vkms. After some bisecting and some study about KMS's
+atomic API, I was able to detect the problem and write a solution for it. It was
+pretty exciting to deal with vkms for the first time and to get some notion
+about the display side of things.
+
+| Patch/Series | Status |
+| :----------- | :----: |
+| [drm/tests: Split drm_test_dp_mst_calc_pbn_mode into parameterized tests](https://patchwork.freedesktop.org/patch/505760/?series=109345&rev=1)           | **Accepted** |
+| [drm/tests: Split drm_test_dp_mst_sideband_msg_req_decode into parameterized tests](https://patchwork.freedesktop.org/patch/505761/?series=109345&rev=1) | **Accepted** |
+| [tests/kms_addfb_basic: Avoid open-coded expressions](https://patchwork.freedesktop.org/series/516047/)                                                  | **Accepted** |
+| [\[PATCH 0/3\] Check for valid framebuffer's format](https://patchwork.freedesktop.org/series/112546/)                                                   | In Review |
+| [drm/vkms: reintroduce prepare_fb and cleanup_fb functions](https://patchwork.freedesktop.org/series/112487/)                                            | **Accepted** |
+
+# Next Steps
+---
+
+A bit different from the end of GSoC, I'm not really sure what are going to be
+my next steps in the next couple of months. The only thing I know for sure is
+that I will keep contributing to the DRM subsystem and studying more about DRI,
+especially the 3D rendering and KMS.
+
+The DRI infrastructure is really fascinating and there is so much to be learn!
+Although I feel that I improved a lot in the last couple of months, I still feel
+like a newbie in the community. I still want to have more knowledge of the DRM
+core helpers and understand how everything glues together.
+
+Apart from the DRM subsystem, I'm also trying to take some time to program more
+in Rust and maybe contribute to other open-source projects, like Mesa.
+
+# Acknowledgment
+---
+
+I would like to thank my great mentors [Melissa
+Wen](https://melissawen.github.io/) and [André
+Almeida](https://andrealmeid.com/) for helping me through this journey. I
+wouldn't be able to develop this project without their great support and
+encouragement. They were an amazing duo of mentors and I thank them for
+answering all my questions and helping me with all the challenges.
+
+Also, I would like to thank the DRI community for reviewing my patches and
+giving me constructive feedback. Especially, I would like to thank Daniel Vetter
+for answering patiently every single question that I had about the debugfs
+clean-up and to thank Jani Nikula, Maxime Ripard, Thomas Zimmermann, Javier
+Martinez Canillas, Emma Anholt, Simon Ser, Iago Toral, Kamil Konieczny and
+many others that took their time to review my patches, answer my questions and
+provide me constructive feedback.
```

Comment:
I dropped minor comments, but all good to me. Thanks for all your work, commitment and insightful questions. Participating in your evolution as a DRM developer has been a pleasure! Keep doing a great work in the DRI community! "Jogue duro"